wellregistry/registry/admin/monitoring_location.py

```diff
@@ -34,15 +218,27 @@ class MonitoringLocationAdminForm(ModelForm):
     """
     Registry admin form.
     """
-
-    # pylint: disable=E1101
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if not self.user.is_superuser:
             user_group = _get_groups(self.user)[0]
             agency_field = ModelChoiceField(queryset=AgencyLookup.objects.filter(agency_cd=user_group),
                                             initial=user_group)
             self.fields['agency'] = agency_field
+            for item in tooltips:
```

Comment:
The code that sets the tooltip needs to be for all users so should be moved out of this if clause.

Reply:
fixed